plugins/modules/zos_copy.py

```diff
@@ -2501,6 +2532,34 @@ def run_module(module, arg_def):
         )
 
     # ********************************************************************
+    # To validate the source and dest are not lock in a batch process by
+    # the machine and not generate a false positive check the disposition
+    # for try to write in dest and if both src and dest are in lock.
+    # ********************************************************************
+    if src_ds_type != "USS" and dest_ds_type != "USS":
+        is_source_lock = data_set_locked(src_name)
+        is_dest_lock = data_set_locked(dest_name)
+        if is_source_lock and is_dest_lock:
+            module.fail_json(
+                msg="DATASETS in lock, unable to access'{0}' and unable to write in'{1}'".format(
```

Comment:
Is "DATASETS" a parameter name here? If not, be careful of using all uppercases in a message, as it could sound angry to users. Use the lowercase two-word phrase "data sets":

> msgs="Data sets locked, unable to access '{0}' and unable to write in '{1}'"...

(Just to confirm -- {0} and {1} will be replaced by the source name and the destination name when users see the message, right?)